themes/default/content/docs/concepts/vs/terraform/_index.md

```diff
@@ -224,6 +224,10 @@ Aliases help facilitate refactoring by allowing you to modify certain properties
 
 Transformations, which are unique to Pulumi, allow you to programmatically set or override the input properties of resources belonging to a particular collection, such as the child resources of a Pulumi component or even all of the resources belonging to a stack. Transformations make it easy to apply consistent settings across your infrastructure without having to manipulate the properties of individual resources. To learn more, see [Transformations](/docs/concepts/options/transformations/) in the Resource documentation.
 
+### Terminology and Commands
```

Comment:
I am wondering if we should just put the chart from that terminology page directly into this page and hide it in an accordion.

Reply:
This is pretty important stuff but its somehow hidden. maybe we need to link to it from above?